### PR TITLE
Feature/authentication

### PR DESCRIPTION
**Pull request by ralfz123**
I'm done with authentication. Now it's your turn to review my code 🔥

### CODE REVIEW

File: client/src/assets/scss/main.scss [hunk 1]
```diff
@@ -1,14 +1,72 @@
...
+
+.gray{
+  background-color: $gray;
+}
+
+.pinkTwo{
+  background-color: $lightPink;
+}
+
+.yellowTwo{
```
Comment (RowinRuizendaal) [inline client/src/assets/scss/main.scss:68]: Next time please use the space bar after the declartion :) .yellow {

File: client/src/services/DataService.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ class DataService {
   }
 
   loginUser(data) {
+    console.log(data);
```
Comment (RowinRuizendaal) [inline client/src/services/DataService.js:17]: might need to remove this later on

**Review by RowinRuizendaal — APPROVED** [review]
Lekker gedaan, had wat kleine opmerkingen (zie discord), deze opmerkingen zijn meer styling dingen :) 👍🏻

**Comment by ralfz123** [comment]
Great, nice to hear! Big thanks

**Pull request merged**